Crafttweaker_Annotation_Processors/src/main/java/com/blamejared/crafttweaker_annotation_processors/processors/document/page/member/static_member/BracketExpressionStaticMethodMember.java

```diff
@@ -0,0 +1,74 @@
+package com.blamejared.crafttweaker_annotation_processors.processors.document.page.member.static_member;
+
+import com.blamejared.crafttweaker_annotation_processors.processors.document.page.comment.DocumentationComment;
+import com.blamejared.crafttweaker_annotation_processors.processors.document.page.member.header.MemberHeader;
+import com.blamejared.crafttweaker_annotation_processors.processors.document.page.type.AbstractTypeInfo;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.PrintWriter;
+
+public class BracketExpressionStaticMethodMember extends StaticMethodMember {
+    
+    private final String bepName;
+    
+    public BracketExpressionStaticMethodMember(String name, MemberHeader header, @Nullable DocumentationComment methodComment, String returntypeInfo, String bepName) {
+        super(name, header, methodComment, returntypeInfo);
+        this.bepName = bepName;
+    }
+    
+    public String getBepName() {
+        return bepName;
+    }
+    
+    @Override
+    protected void writeExampleBlockContent(PrintWriter writer, AbstractTypeInfo ownerType) {
+        writeBepExamples(writer);
+        super.writeExampleBlockContent(writer, ownerType);
+    }
+    
+    private void writeBepExamples(PrintWriter writer) {
+        final int numberOfUsableExamples = header.getNumberOfUsableExamples();
+        if(numberOfUsableExamples < 1) {
+            return;
+        }
+        
+        for(int exampleIndex = 0; exampleIndex < numberOfUsableExamples; exampleIndex++) {
+            writeBepExample(writer, exampleIndex);
+        }
+        writer.println();
+    }
+    
+    private void writeBepExample(PrintWriter writer, int exampleIndex) {
+        final String example = getExampleArgument(exampleIndex);
+        writer.printf("<%s:%s>%n", bepName, example);
```

Comment:
Does this not need to be `&lt;` and `&gt;`?

Reply:
This prints an example, and examples are inside a code block